haskell-indentation.el

```diff
@@ -87,18 +100,13 @@
 It rebinds RET, DEL and BACKSPACE, so that indentations can be
 set and deleted as if they were real tabs."
   :keymap haskell-indentation-mode-map
-  (kill-local-variable 'indent-line-function)
```

Comment:
This `kill-local-variable`s should stay here. Reason: minor modes may be switched on and off, this code path is for switching off (although it is executed always).